resource-aggregate/service/grpcApi.go

```diff
@@ -201,9 +201,19 @@ func (r RequestHandler) UpdateResource(ctx context.Context, request *commands.Up
 	if err != nil {
 		log.Errorf("cannot publish resource content update events: %v", err)
 	}
+
+	var validUntil int64
+	for _, e := range events {
```

Comment:
Why iterating and not taking events[len(events)-1] ?

Reply:
in cancellation MR it can create a snapshot after the event so I cannot use it. And in the worst case, it contains 2 events.